<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminactions</class>
 <widget class="QMainWindow" name="adminactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
	background-color: #7fcdee;
}

QLabel {
	color: #07407b;
}

QLineEdit {
	border: 2.5px solid #f7931e;
	border-radius: 5px;
	padding: 7px 20px;
}

QSpinBox {
	border: 2.5px solid #f7931e;
	border-radius: 5px;
	padding: 7px 20px;
}

QSpinBox::up-button {
	subcontrol-origin: border;
	subcontrol-position: top right;
	width: 40px;
	height: 18px;
}

QSpinBox::down-button {
	subcontrol-origin: border;
	subcontrol-position: bottom right;
	width: 40px;
	height: 18px;
}

QPushButton {
	color: #ffffff;
	font-weight: bold;
	background-color: #f7931e;
	border-radius: 5px;
	padding: 7px 20px;
}

QPushButton::hover {
	color:  #f7931e;
	font-weight: bold;
	background-color: #ffffff;
}

#FoodStockList {
	font-size: 16px;
	background-color: #07407b;
	border: 2.5px solid #f7931e;
	border-radius: 15px;
}

#FoodStockList::item {
	color: #ffffff;
}

#FoodStockList::item::selected {
	color: #f7931e;
}

#AdminLabel {
	color: #07407b;
	font-weight: bold;
	font-size: 20px;
}

#SelectedLabel {
	color: #07407b;
	font-weight: bold;
	font-size: 20px;
}

#AdminBox {
	background-color: #07407b;
	border: 2.5px solid #f7931e;
	border-radius: 15px;
}

#AdminBox QLabel {
	color: #ffffff;
	font-size: 14px;
	font-style: italic;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="AdminLabel">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Admin Actions</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Check Notifications</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LogoutButton">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="FoodStockList"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignTop">
         <widget class="QPushButton" name="HealthSafetyButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Health and Safety Letter</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="UserManagementButton">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>User Management</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPlainTextEdit" name="ReceiptTextEdit">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>Receipt:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="SelectedLabel">
      <property name="text">
       <string>Currently Selected:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="AdminBox" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="AddAmountLabel">
               <property name="text">
                <string>Amount to Add:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="AddAmountSpinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>500</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="ReorderButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Add to Order</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SubmitButton">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Submit Order</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
